iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -204,9 +239,11 @@ template <uint64_t Capacity, typename ReturnType, typename... Args>
 template <typename CallableType>
 inline void storable_function<Capacity, signature<ReturnType, Args...>>::destroy(storable_function& f) noexcept
 {
-    if (f.m_callable)
+    if (f.m_callable != nullptr)
```

Comment:
Being explicit here may be useful, but IMO should be optional. The meaning is well-defined to my knowledge.